### ISSUE TITLE
Fix Error Messaging

### ISSUE THREAD

**Issue by RyanThomas73** [ds300/patch-package#131]
Previously, using `patch-package v5.1.1`, errors applying patches provided detailed messaging in the format: 

```
**ERROR** Failed to apply patch for package <package name>

... DETAILED INFO AND SUGGESTED RESOLUTION ...
```

When testing using the `6.0.5` version, the only error message displayed (shown) below provides very little information. The error no longer provides the package name and/or patch file path that it was attempting to parse and it does not provide clear instructions on what the problem is.

```
Error: hunk header integrity check failed
    ... <stack trace> ...
```

**Comment by ds300** [comment]
This is great feedback thanks! I'll look into fixing this asap.

**Comment by ds300** [comment]
This is fixed now in v6.0.6 :tada:

Here's an example error message https://github.com/ds300/patch-package/blob/510df53e8c840df240b797739abe8fa729bb69ec/src/patch/read.test.ts#L37-L51
